### ISSUE TITLE
[Bug]: Shirts no longer attach to skirts/trousers/so on.

### ISSUE THREAD

**Issue by Melariara** [Aurorastation/Aurora.3#18392]
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Horizon

### Describe the issue

In the loadout and as well as when you spawn in, characters shirts no longer automatically are worn, leading to characters spawning in with no shirts, but things such as waistcoats attached to them instead.

![f498c033ea483e1020e05dd9b916a631](https://github.com/Aurorastation/Aurora.3/assets/60306280/0d67ad69-1650-4d5c-a1fa-9a93084a39b5)
![4545387094f1c9912d2312097208a8ed](https://github.com/Aurorastation/Aurora.3/assets/60306280/14d117de-0d86-4b12-a52f-620572bfe832)
(This is how she spawns in as well)

### How to reproduce

1. Have a shirt, have some pants.
2. Check loadout, be missing shirt.
3. Spawn in like this.

**Comment by FluffyGhoster** [comment]
![image](https://github.com/Aurorastation/Aurora.3/assets/65877598/4391cef9-fa1b-49da-8bae-1c9f581979f7)
![image](https://github.com/Aurorastation/Aurora.3/assets/65877598/ce590d78-3100-4a61-9c35-52f80195dabc)

Unable to reproduce

**Comment by zagoon** [comment]
This was a result of the testmerge a couple days back that messed up accessories. Updating one of the item's properties (color, name, whatever) and saving it fixed the issue on my end.

**Comment by SleepyGemmy** [comment]
can replicate.
one of my older loadouts experience this issue, where the shirt is missing. it appears again if i remove an accessory.

**Comment by FluffyGhoster** [comment]
> can replicate. one of my older loadouts experience this issue, where the shirt is missing. it appears again if i remove an accessory.

If you change any setting of the loadout, can you still replicate it?

**Comment by SleepyGemmy** [comment]
> If you change any setting of the loadout, can you still replicate it?

i'll check tomorrow.

**Comment by SleepyGemmy** [comment]
> If you change any setting of the loadout, can you still replicate it?

it seems to "refresh" and show the accessories properly by just selecting a random accessory then unselecting it.
if i then save the loadout and reload, switch slots, etc, the accessories still show as expected.
i'd consider this the solution and it's just a visual bug.

TLDR: select a random accessory, remove it, then save the slot. it should now show as normal.
